audio: th1520: enable soundcard feature

leeyong5866 opened the pull request:
enable TH1520-Sound-Card feature

	simple-audio-card,dai-link@0 {          /* I2S - HDMI*/
         reg = <0>;
         format = "i2s";
         cpu {
             sound-dai = <&ap_i2s 1>;
         };
         codec {
             sound-dai = <&hdmi_codec>;
         };
     };
     simple-audio-card,dai-link@1 {          /* I2S - AUDIO SYS CODEC 7210*/
         reg = <1>;
         format = "i2s";
         cpu {
             sound-dai = <&i2s1 0>;
         };
         codec {
             sound-dai = <&es7210_audio_codec>;
         };
     };
     simple-audio-card,dai-link@2 {          /* I2S - AUDIO SYS CODEC 8156*/
         reg = <2>;
         format = "i2s";
         cpu {
             sound-dai = <&i2s1 0>;
         };
         codec {
             sound-dai = <&es8156_audio_codec>;
         };
     };

leeyong5866 commented:
@RevySR 目前驱动中（甚至 arch 中）依然在用 LINUX_VERSION_CODE，这一项 check 没有必要，请确认

RevySR commented:
> @RevySR 目前驱动中（甚至 arch 中）依然在用 LINUX_VERSION_CODE，这一项 check 没有必要，请确认

没有必要 合并到内核树中内核版本是确定的 不需要版本检查做兼容行为

Commit:
audio: th1520: enable soundcard feature

enable TH1520-Sound-Card feature
    simple-audio-card,dai-link@0 {          /* I2S - HDMI*/
        reg = <0>;
        format = "i2s";
        cpu {
            sound-dai = <&ap_i2s 1>;
        };
        codec {
            sound-dai = <&hdmi_codec>;
        };
    };
    simple-audio-card,dai-link@1 {          /* I2S - AUDIO SYS CODEC 7210*/
        reg = <1>;
        format = "i2s";
        cpu {
            sound-dai = <&i2s1 0>;
        };
        codec {
            sound-dai = <&es7210_audio_codec>;
        };
    };
    simple-audio-card,dai-link@2 {          /* I2S - AUDIO SYS CODEC 8156*/
        reg = <2>;
        format = "i2s";
        cpu {
            sound-dai = <&i2s1 0>;
        };
        codec {
            sound-dai = <&es8156_audio_codec>;
        };
    };

Signed-off-by: David Li <davidli.li@linux.alibaba.com>

leeyong5866 commented:
> > @RevySR 目前驱动中（甚至 arch 中）依然在用 LINUX_VERSION_CODE，这一项 check 没有必要，请确认
> 
> 没有必要 合并到内核树中内核版本是确定的 不需要版本检查做兼容行为

已修复
